src/common/api/swagger.yaml

```diff
@@ -1002,6 +1002,11 @@ paths:
           type: string
           format: uuid
           required: true
+        - in: query
+          name: timestamp
+          type: integer
+          description: The time on the host as seconds since the Unix epoch.
+          required: false
```

Comment:
Please make sure you are only updating the parts of `swagger.yaml` that are relevant to this issue.

Reply:
True, thanks for pointing that out